src/test/app/case/types/alert/alert.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai';
+import { Alert } from '../../../../../main/app/case/types';
```

Comment:
once my PR goes in, you can simplify above as import { Alert, SubTask, TaskList, Task, TaskStatus } from 'case/types';